test/test_maskedtensor.py

```diff
@@ -64,13 +64,9 @@ def _compare_mts(mt1, mt2, rtol=1e-05, atol=1e-08):
     if not _tensors_match(a, b, exact=False, rtol=rtol, atol=atol):
         raise ValueError("The data in MaskedTensor mt1 and MaskedTensor mt2 do not match")
 
-def _make_tensor_mask(shape, device):
-    return make_tensor(
-        shape, device=device, dtype=torch.bool, low=0, high=1, requires_grad=False
```

Comment:
It doesn't matter anymore, but `high=1` was wrong here, since the high value is exclusive.

Reply:
Not sure I understand. `torch.bool` is hardcoded here. Are are you just saying that `high` is also exclusive for integral dtypes? If so, that is true, but not the whole picture. `high` in `make_tensor` is exclusive for all dtypes.

Reply:
Nvm, we have had the same bug for integer dtypes in past. I was just surprised that masked tensors still had it. It basically meant that all related tests were using degenerate masks (all false).

Reply:
> all related tests were using degenerate masks (all false).

No, the tests were fine. `torch.testing.make_tensor` ignored `low` and `high` for `torch.bool`. #96332 in this stack fixes that, which also surfaces this error.